moves/tests.py

```diff
@@ -1,3 +1,81 @@
+import pdb
+import json
+import datetime
+from django.utils import timezone
+from django.test import override_settings
 from django.test import TestCase
+from .models import Move, Spot
 
 # Create your tests here.
+class IndexTests(TestCase):
+    def test_uuid_is_set_after_navigating_to_the_index_page(self):
+        self.assertIsNone(self.client.session.get('user_uuid'))
+        self.client.get('/')
+        self.assertTrue(bool(self.client.session.get('user_uuid')))
```

Comment:
I told you to do with it assertIsNone/assertTrue, but after thought I think that `self.assertNotIn('user_uuid', self.client.session)` and `self.assertIn('user_uuid', self.client.session)` would be more semantically meaningful/correct because that's really what happening is here.